rosidl_typesupport_connext_c/bin/rosidl_typesupport_connext_c

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+
+from rosidl_cmake import read_generator_arguments
+from rosidl_parser import UnknownMessageType
+from rosidl_typesupport_connext_c import generate_typesupport_connext_c
+
+
+def is_valid_file(parser, file_name):
+    if not os.path.exists(file_name):
+        parser.error("File does not exist: '{0}'".format(file_name))
+    file_name_abs = os.path.abspath(file_name)
+    if not os.path.isfile(file_name_abs):
+        parser.error("Path exists but is not a file: '{0}'".format(file_name_abs))
+    return file_name
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description='Generate the C interfaces for Connext.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '--generator-arguments-file',
+        required=True,
+        help='The location of the file containing the generator arguments')
+    parser.add_argument(
+        '--dds-interface-base-path',
+        required=True,
+        help='The base location of the DDS interface files')
+    parser.add_argument(
+        '--idl-pp',
+        required=True,
+        help='The location of the IDL preprocessor')
```

Comment:
I realized that we can remove these two options from the script because they are only used by the part we delegate to the C++ generator. Here is the similar change I just made for opensplice: https://github.com/ros2/rmw_opensplice/commit/893bb5baf7b66a440bfe04b1dbee66a5fbc0b327